packages/embed-widget/src/index.tsx

```diff
@@ -5,7 +5,6 @@ import { store } from '@deephaven/redux';
 import '@deephaven/components/scss/BaseStyleSheet.scss';
 import { LoadingOverlay, preloadTheme } from '@deephaven/components';
 import { ApiBootstrap } from '@deephaven/jsapi-bootstrap';
-import './index.scss';
```

Comment:
I can't remember why this was removed to begin with. I would just add it back as I'm not sure if it does anything, so not sure it can safely be removed

Reply:
It was because it was not providing styling: https://github.com/deephaven/web-client-ui/pull/2023#discussion_r1605666964

Reply:
Right but that was after removing the style in the index file in this PR. I wasn't sure why the style was removed.

The only thing I can think of would be this is needed for some reason in jupyter, but I'm not sure.

Reply:
Okay yeah I can add it back, doesn't do anything for my logic so don't really mind either way.